changelog/fragments/1693920861-netinfo-manifests.yaml

```diff
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: enhancement
+
+# Change summary; a 80ish characters long description of the change.
+summary: netinfo-manifests
```

Comment:
You should explain what the actual change does that users can understand it, this is a change in the data that is collected so we shouldn't be brief.

Reply:
Thanks Craig, see update [debff38](https://github.com/elastic/elastic-agent/pull/3354/commits/debff38053b6c1eaefc4a996aced07e0247762be)